nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -273,8 +280,24 @@ let
     export with_serial
     clear
     set timeout=10
+
+    # This message will only be viewable when "gfxterm" is not used.
+    echo "Please press 't' to show the boot menu on this console"
+
```

Comment:
With more testing I see that this is shown for just enough time on the device's display (e.g. pinebook pro).

This could cause confusion. I'm thinking about rephrasing it to something like:

```
Loading graphical boot menu...
Press 't' to show use text boot menu on this text console...
```

This should make it more obvious that, even though it's showing for only 1~2 seconds, that something is loading, rather than just a prompt to press 't'.